src/systemTest/java/com/flightstats/hub/callback/model/RequestObject.java

```diff
@@ -0,0 +1,13 @@
+package com.flightstats.hub.callback.model;
+
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class RequestObject {
+    //webhook name
+    private String name;
```

Comment:
+1, though it probably means an annotation to make it serialize "name" as the field since I'm guessing that "name" is part of the spec'd callback api?

Reply:
I think the callback API is whatever we define it as, but I might be mistaken.

Reply:
If this is something that your customers see, then the current "name" is a contract you wouldn't want to break. Someone creating a request is going to be creating it as "name".

Reply:
Yes. the it comes in as "name" in the POST request body.

Reply:
Oh, I see. For some reason I thought this was something that was internal to our callback server and not what the webhook returned to the callback. Duh.